src/dash/DashMetrics.js

```diff
@@ -33,6 +33,7 @@ import ManifestModel from '../streaming/models/ManifestModel';
 import DashManifestModel from './models/DashManifestModel';
 import FactoryMaker from '../core/FactoryMaker';
 import * as MetricsList from './constants/DashMetricsList';
+const round10 = require('round10').round10;
```

Comment:
Can I asked why we are using require in this case vs import as we do with codem-isoboxer?